### ISSUE TITLE
Sign in doesn't return userId 

### ISSUE THREAD

**Issue by ninox14** [vitaly-sazonov/kanban-rest#4]
How are you supposed to get it if you logged out ?

**Comment by ninox14** [comment]
#6  Pull request with fixes

**Comment by vitaly-sazonov** [comment]
1. why added cors?
2. explain more about logout

**Comment by ninox14** [comment]
1. Why shouldn't it be enabled in the first place? Coz its prevents you from working with it even running locally.
2. For example: 
 User registered and immediately logged in, than he logged off and we deleted his token from localStorage.
 Another user logs in (with different id) gets token and we set it up in localStorage. Now this user wants to use an application so we need his userId to send request to the server, how do we get it ? 

Just think about flow of authentication of the client application with current implementation and you will realize how bad this server actually is.
